### PR TITLE
RSSAgent should record which feed errored and keep going on the other ones

### LINKED ISSUE #849: Add Massive RSS feed for filtering
Hi,

As Yahoo Pipes is closing, I'm looking for using Huginn as a new platform. However, I would like to know if it's possible to add a lot of RSS feed (because currently in the RSSagent you can only add one url) ?

Regards

### PR DESCRIPTION
**Pull request by cantino**
@dsander @knu Do you agree with the change that the RSSAgent, when given multiple feeds, should log errors for failed ones but keep going on the others?

Fixes #849

### CODE REVIEW

**Comment by knu** [comment]
@cantino Agreed!

File: Gemfile.lock [hunk 1]
```diff
@@ -580,6 +580,3 @@ DEPENDENCIES
   weibo_2!
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
-
```
Comment (cantino) [inline Gemfile.lock]: How do I stop this from being removed?  I updated bundler and now `bundler --version` reports `Bundler version 1.10.6`, but whenever I run `bundle`, this happens.  Oddly, even editing the Gemfile.lock and adding it back in seems to be unable to save the change, like some background process keeps removing it.

Comment (dsander) [inline Gemfile.lock]: Is it possible you have a old spring instance running?

Comment (cantino) [inline Gemfile.lock]: Yes!

Ugh, I hate spring.

Comment (virtadpt) [inline Gemfile.lock]: -----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On 08/01/2015 01:16 PM, Andrew Cantino wrote:

> Ugh, I hate spring.

Would some benefit be derived from replacing Spring with something
like Zeus or Commands?  Or would they not be a good fit for Huginn's
architecture?

- -- 
The Doctor [412/724/301/703/415] [ZS]

PGP: 0x807B17C1 / 7960 1CDC 85C9 0B63 8D9F  DD89 3BD8 FF2B 807B 17C1
WWW: https://drwho.virtadpt.net/

"Shockingly, that didn't work! I can't imagine why!" --Adam Savage

-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJVvmOKAAoJED1np1pUQ8RkgYwP/1/zW1TONuf9vMpQhJFO/Yeq
EQDejkIBw0O5Xy99BRulcAQWdBNH9or6jh41vpuSENVyVRpx61d7KseHGz5+tWh0
7PQ61rdM47tjs9ZULTmHbSJKHWH2Qsok7WDqzsAJ6QV3XQNffAQ2L+ilA3EV6ir/
XtnL7inTnGvdhroT366+K+6aD6O12TBnl0+zEi3/PVMmS4YDIJQSM6MP2W61M4Zx
iOCOwfQ1hxfFuEUXbFM0ZNtWTT/lKa38Ceuo/cRgE2viqwGvMqX1gvUNXXCWi2tg
AvG+FawCywEuD4v7di4tb2F+FhgEfnwJj5TZTYepe2IExkT9HRX02X8cp93Gww4s
ryUonvWvd88HojHs9yRg3cvqvl/9fd9rnSJlrxajgt3L9G28+WgZm985jVDr887U
nk3gLCYZ+79JMyleZyiakASqRPdhHuKyX+ZoepwM92nyry856l2Gew+m6Q6KZsRk
5uoWQLvABqSl2AluTR/EEk6kRbsBdnenO9jHENkSuOtXOASjgzYYC9ButoCj7KRi
uhJRGQPmZuOM1P7NVCfzxqp+VcjosBHB7bI9I7jTt8vpq62W2t9iN08KBqTKlebZ
igboh10Ps1OytG7rVAnRUrQXHLFskCqdD4nKAT1kGoBlf8hR3SY/HWr6eQ2EVOnM
GOFls4bcdWT8mK/aU5mx
=NfSI
-----END PGP SIGNATURE-----

**Pull request merged**